pkg/kor/all.go

```diff
@@ -316,7 +316,34 @@ func GetUnusedAll(filterOpts *filters.Options, clientset kubernetes.Interface, a
 		fmt.Printf("err: %v\n", err)
 	}
 
-	unusedAll := unusedAllNamespaced + unusedAllNonNamespaced
+	unusedAll := make(map[string]interface{})
 
-	return unusedAll, nil
+	if outputFormat != "json" {
```

Comment:
We support yaml as a conversion of the json so we'd need to support that in the condition

Reply:
When adding up both YAML results from the two called functions, they are generated separately and aggregated properly, in case of JSON they are not aggregated properly since each result is wrapped under {}.